lib/types.ts

```diff
@@ -233,4 +248,5 @@ export type Verification = {
   cancellation_details?: string;
   loan_id?: string;
   reports?: Report[];
+  verifier_type: VERIFIER_TYPE;
```

Comment:
is this a backwards-incompatible change?

Reply:
depends on your definition. it adds a new required field to a read model. doesn't break old readers, but if customers are building their own verification models (ex for testing) it will break.

wdyt?

Reply:
if it's read it should be fine to extend the return type's structure. I thought this might be adding a required parameter to a write API.

Reply:
I see RequestVerificationsCreate's change is `?`, so lgtm